src/Movement/Kinematics/HangprinterKinematics.cpp

```diff
@@ -187,48 +238,73 @@ bool HangprinterKinematics::Configure(unsigned int mCode, GCodeBuffer& gb, const
 		{
 			Kinematics::Configure(mCode, gb, reply, error);
 			reply.lcatf(
+				"P:Print radius: %.1f\n"
 				"A:%.2f, %.2f, %.2f\n"
 				"B:%.2f, %.2f, %.2f\n"
 				"C:%.2f, %.2f, %.2f\n"
-				"D:%.2f, %.2f, %.2f\n"
-				"P:Print radius: %.1f",
+				"D:%.2f, %.2f, %.2f\n",
+				(double)printRadius,
 				(double)anchors[A_AXIS][X_AXIS], (double)anchors[A_AXIS][Y_AXIS], (double)anchors[A_AXIS][Z_AXIS],
 				(double)anchors[B_AXIS][X_AXIS], (double)anchors[B_AXIS][Y_AXIS], (double)anchors[B_AXIS][Z_AXIS],
 				(double)anchors[C_AXIS][X_AXIS], (double)anchors[C_AXIS][Y_AXIS], (double)anchors[C_AXIS][Z_AXIS],
-				(double)anchors[D_AXIS][X_AXIS], (double)anchors[D_AXIS][Y_AXIS], (double)anchors[D_AXIS][Z_AXIS],
-				(double)printRadius
+				(double)anchors[D_AXIS][X_AXIS], (double)anchors[D_AXIS][Y_AXIS], (double)anchors[D_AXIS][Z_AXIS]
 				);
+			if (numAnchors > 4)
+			{
+        			reply.lcatf("E:%.2f, %.2f, %.2f\n",
```

Comment:
I don't know, I tried to look for more documentation on lcatf, presumably it has something to do with the typical cat in unix.
But without reading the documentation, how could I know?
Where is it?
Can we write unit tests too as part of the documentation?
Where would it be the right place to place tests about kinematics?
Out of context, thank the duet that I don't have to use USB anymore because there's wifi support, even though I prefer cable in the long term.
That's besides the point. The web interface is so much better than pronterface...
Sorry, too much talking, not enough fixing the initial nit/complain.

Reply:
Yep, you were right about the limit size. How do we solved this? I documented it better in the main posrt, I'm going to mark this comment as solved because I think I solved the catf part.
Feel free to open a new one if I didn't solved it or properly documented it.
Better documentation always welcomed.